.readthedocs.yml

```diff
@@ -0,0 +1,2 @@
+build:
+    image: latest
```

Comment:
@hj @michalholasek Here's a trick to use readthedocs image which contains jsdoc and node, npm. I noticed that Dredd is installing node itself, you can simplify Dredd readthedocs configuration by using this method instead.

NOTE, as I am documenting my dependencies, all I needed to do is `npm install` inside docs/conf.py

Reply:
This is awesome!